ndcube/utils/wcs.py

```diff
@@ -477,3 +478,28 @@ def compare_wcs_physical_types(source_wcs, target_wcs):
     target_wcs = get_low_level_wcs(target_wcs, 'target_wcs')
 
     return source_wcs.world_axis_physical_types == target_wcs.world_axis_physical_types
+
+
+def is_wcs_2d_celestial(wcs):
```

Comment:
Instead of returning just a boolean, do we want this function to return/raise more details like whether or not the WCS is 2D, or if it is then are those axes celestial?

Reply:
Okay, thanks! I'll move that check outside.